statsproduct.php

```diff
@@ -384,9 +384,8 @@ public function setOption($option, $layers = 1)
         $date_between = $this->getDate();
         switch ($this->option) {
             case 1:
-                $this->_titles['main'][0] = $this->trans('Popularity', array(), 'Modules.Statsproduct.Admin');
-                $this->_titles['main'][1] = $this->trans('Sales', array(), 'Admin.Global');
-                $this->_titles['main'][2] = $this->trans('Visits (x100)', array(), 'Modules.Statsproduct.Admin');
+                $this->_titles['main'][0] = $this->trans('Sales', array(), 'Admin.Global');
+                $this->_titles['main'][1] = $this->trans('Views (x100)', array(), 'Modules.Statsproduct.Admin');
```

Comment:
This wording doesn't look like it appears in the catalog. If It absolutely needs to be changed, then the module compatibility will have to be upped to 1.7.5.0 so that the wording can be translated.

Reply:
Hi Pablo, the wording is the most minimal part... the functionality was corrected.

Reply:
i apologize for the slow response, I did not receive the email notification you had replied